src/style.css

```diff
@@ -0,0 +1,160 @@
+/* #region block: cards */
+
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  justify-content: center;
+
+  width: 198px;
+  height: 404px;
+
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  padding: 33px 19px 0;
+}
+
+.card__name {
+  margin: 40px 16px 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__productCode {
+  margin: 0 84px 16px 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.card__starbox {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin: 0 16px;
+}
+
+.card__starbox-reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.card__pricebox {
+  margin: 24px 16px 16px;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__pricebox-price {
+  margin: 0;
+  padding: 0;
+
+  width: 31px;
+  height: 18px;
+
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.card__pricebox-number {
+  margin: 0;
+  padding: 0;
+
+  width: 50px;
+  height: 18px;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__buylink {
+  width: 27px;
+  height: 16px;
+
+  margin: 0 16px 16px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  text-transform: uppercase;
+  text-decoration: none;
+
+  color: #fff;
+
+  padding: 12px 70px 12px 69px;
```

Comment:
you dont need to use exact paddings for alignment text, find another more flexible solution)